Documentation/Configuration.md

```diff
@@ -54,7 +54,7 @@ top-level keys and values:
 
 *   `lineBreakBeforeEachArgument` _(boolean)_: Determines the line-breaking
     behavior for generic arguments and function arguments when a declaration is
-    wrapped onto multiple lines. If true (the default), a line break will be
+    wrapped onto multiple lines. If true, a line break will be
```

Comment:
Can you move the "(the default)" part below to follow "If false", and rewrap this paragraph as needed so that it fills up to 80 columns?

Reply:
Sure!